src/quo2/components/inputs/address_input/view.cljs

```diff
@@ -0,0 +1,161 @@
+(ns quo2.components.inputs.address-input.view
+  (:require [react-native.core :as rn]
+            [react-native.clipboard :as clipboard]
+            [quo2.theme :as quo.theme]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as icon]
+            [quo2.components.buttons.button.view :as button]
+            [quo2.components.inputs.address-input.style :as style]
+            [utils.i18n :as i18n]
+            [reagent.core :as reagent]))
+
+(def ^:private ens-regex #"^(?=.{5,255}$)([a-zA-Z0-9-]+\.)*[a-zA-Z0-9-]+\.[a-zA-Z]{2,}$")
+
+(defn- icon-color
+  [blur? override-theme]
```

Comment:
we don't want to have the concept of an `override-theme`, whatever theme value is in the component is just the `theme`

Reply:
Yeah, you are right, I changed it to `theme`